specifications/catalog/catalog.binding.https.md

```diff
@@ -1,20 +1,20 @@
 # Catalog HTTPS Binding {#catalog-http}
 
-This specification defines a RESTful API over HTTPS for the [=Catalog Protocol=].
-
 ## Introduction
 
+This binding defines a RESTful Application Programming Interface (API) over HTTPS for the [=Catalog Protocol=].
+
 ### Prerequisites
 
 1. The `<base>` notation indicates the base URL for a [=Catalog Service=] endpoint. For example, if the base [=Catalog=]
    URL is `api.example.com`, the URL `https://<base>/catalog/request` will map
    to `https//api.example.com/catalog/request`.
 
-2. All request and response [=Messages=] must use the `application/json` media type.
+2. All request and response [=Messages=] _MUST_ use the `application/json` media type.
 
 ### Catalog Error
 
-In the event of a request error, the [=Catalog Service=] must return an appropriate HTTP code and
+In the event of a request error, the [=Catalog Service=] _MUST_ return an appropriate HTTP code and
```

Comment:
respec recognizes keywords from RFC2119/RFC8174. Doing this explicitly isn't harmful but redundant.